elibrary-hibernate/src/main/resources/db-changelog-master.xml

```diff
@@ -54,5 +54,6 @@
 	<!-- it is possible to run DML on local: -->
 	<!--   - liquibase plugin part - db configuration - should be commented in pom.xml; -->
 	<!--   - all data should be truncated; -->
-	<!--   - last version changelog should be applied using mvn liquibase:update using dev profile. -->
```

Comment:
Please remove it as it's a local change. 🗑️

Reply:
I did it.